src/commonMain/kotlin/com/eiknat/lemmyclient/apipublic/user/Register.kt

```diff
@@ -0,0 +1,15 @@
+package com.eiknat.lemmyclient.apipublic.user
+
+
+interface RegisterRequest {
+	val username: String
+	val email: String?
+	val password: String
+	val passwordVerification: String
+	val admin: Boolean
+	val showNsfw: Boolean
+}
+
+interface RegisterResponse {
+	val jwt: String
+}
```

Comment:
Some classes are using tabs, some spaces. I feel this should be more consistent.

Reply:
my ide broke ._.